client/components/formatted-date/docs/example.js

```diff
@@ -50,36 +62,24 @@ class FormattedDateExample extends PureComponent {
 	render() {
 		return (
 			<div>
-				<label>
+				<FormLabel>
 					Date
 					<br />
-					<input
-						type="text"
-						name="theDate"
+					<FormTextInput
 						onChange={ this.handleDateChange }
-						defaultValue={ this.state.currentDateString }
+						value={ this.state.currentDateString }
 					/>
```

Comment:
I removed it because `name` is basically unnecessary here, so it's best if we remove it. Maybe we should clean those other instances up too, but I'll leave that for another time 😉